test/fail_compilation/failcstuff2.c

```diff
@@ -97,15 +93,6 @@ void test22068()
     --(long long) var;
 }
 
-#line 200
-/***************************************************/
-// https://issues.dlang.org/show_bug.cgi?id=21992
-void test21992(int var)
-{
-    var = (var) ~ 1234;
-    var = (var) ! 1234;
-}
-
```

Comment:
This code still shouldn't be compilable.

Reply:
Right, but it produces different error messages and aborts the compilation. Besides, the special case it was testing no longer exists.

Reply:
Ah, then failcstuff1.c is the place to fling this then for parse-time errors.

Reply:
But the test has become redundant.